src/core/InputBase.tsx

```diff
@@ -33,6 +33,7 @@ const InputAdornment: SFC<IProps> = ({
     ...otherProps
 }) =>
     <MuiInputBase
+        autoComplete="off"
```

Comment:
`autoComplete={ autoComplete }`

Reply:
same to `TextField.tsx` file

Reply:
`'autoComplete' is assigned a value but never used.eslint(@typescript-eslint/no-unused-vars)`